db/migrate/20230209222225_add_linkable_index_to_sidebar_section_links.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddLinkableIndexToSidebarSectionLinks < ActiveRecord::Migration[7.0]
+  def change
+    add_index :sidebar_section_links, %i[linkable_type linkable_id]
```

Comment:
I think since it's already a huge table you need to do this concurrently, see:

https://github.com/discourse/discourse/blob/c19394c18463d0a18fab4c368ea69292f033d161/plugins/chat/db/migrate/20221219082223_add_index_to_chat_messages.rb

Reply:
Also adding `disable_ddl_transaction!`